LinqToSqlAdapter.UnitTests/LinqToSqlAdapter.UnitTests.csproj

```diff
@@ -1,199 +1,28 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="Current">
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <ProjectType>Test</ProjectType>
+    <UsesMocking>true</UsesMocking>
+  </PropertyGroup>
+
+  <Import Project="..\Shared.build.props" />
   <Import Project="..\packages\NUnit.3.12.0\build\NUnit.props" Condition="Exists('..\packages\NUnit.3.12.0\build\NUnit.props')" />
```

Comment:
@a-ctor we're using this import for some nunit2/nunit3 magic. we need at least switch this to use a nuget-package-property instead of hardcoding the path.